tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1459,4 +1459,62 @@ public function theResponseShouldContainTheFollowingDriveInformation(TableNode $
 			throw new Error('Response is not an array or the array does not consist key "drive"');
 		}
 	}
+
+	/**
+	 * @When user :user gets all applications using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userGetsAllApplicationsUsingTheGraphApi(string $user) {
+		$response = GraphHelper::getApplications(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user),
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @param TableNode $table
+	 *
+	 * @Then the user retrieve API response should contain the following applications information:
+	 *
+	 * @return void
+	 */
+	public function theResponseShouldContainTheFollowingApplicationsInformation(TableNode $table): void {
+		Assert::assertIsArray($responseArray = ($this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse()))['value'][0]);
+		foreach ($table->getHash() as $row) {
+			$key = $row["key"];
+			if ($key === 'id') {
+				Assert::assertTrue(GraphHelper::isUUIDv4($responseArray[$key]), __METHOD__ . ' Expected id to have UUIDv4 pattern but found: ' . $row["value"]);
```

Comment:
```suggestion
				Assert::assertTrue(
				    GraphHelper::isUUIDv4($responseArray[$key]),
				    __METHOD__ . ' Expected id to have UUIDv4 pattern but found: ' . $row["value"]
				);
```
just to make the review easier